report_pre_thesis/src/sections/methodology.tex

```diff
@@ -1,8 +1,191 @@
+
+\subsubsection{Data Preprocessing}
+Similar to the original pipeline, we have used the 408 samples from the ChemCam calibration dataset.
+Before performing ICA, we have to preprocess this data, which is illustrated in figure \ref{fig:ica_data}.
+As in section \ref{sec:...}, we start by masking the data such that the spectral-range edges are removed. %TODO: update reference when PLS section is done
+Then we normalize the data --- once using Norm1 and once using Norm3.
+This gives us two datasets with normalized data.
+Both of these datasets are then used for ICA separately because, as \citeauthor{cleggRecalibrationMarsScience2017} found, some of the oxides are better modeled with data normalized using Norm1, while others are better modeled with data normalized using Norm3.
+The independent components from each of these two datasets are saved to a file and used for regression later.
+These steps are identical to the original pipeline described by \citeauthor{cleggRecalibrationMarsScience2017}.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.2675\textwidth]{images/ica_data.png}
+	\caption{The ICA phase in our recreation of the pipeline.}
+	\label{fig:ica_data}
+\end{figure}
+
+However, we do not weigh by the inverse of the instrument response function (IRF) as mentioned in \citeauthor{cleggRecalibrationMarsScience2017} before normalizing.
+The reason for this is that the practical implications of weighing by the inverse of the IRF has been subject to criticism by some of the original authors since it effectively reintroduces the noise that was removed by applying the IRF in the first place.
+As previously mentioned, the IRF is applied specifically to remove noise from the data during the processing of the EDR to CCS data.
+Therefore, weighing by the inverse of the IRF seems counterproductive based on this information.
+
+In addition, because \citeauthor{cleggRecalibrationMarsScience2017} does not mention how each of the five datasets are used for each target, we have decided to only use the first dataset for each target.
+This likely does not produce as accurate results as using all five datasets would, since we do not get a full representation of the sample that was shot at by the laser, instead only getting a partial representation from a single location.
+Nevertheless, to avoid deviating significantly from \citeauthor{cleggRecalibrationMarsScience2017}'s methodology or making unfounded assumptions about their data processing, we have opted for this more conservative approach.
+Our goal is to test each of the components of the pipeline individually rather than trying to reproduce the results to absolute perfection.
+
+Similarly, in our recreation of the pipeline, we have not incorporated outlier detection in the ICA part of the pipeline primarily because \citeauthor{cleggRecalibrationMarsScience2017} does not describe their implementation sufficiently enough for precise replication.
```

Comment:
Better?
```suggestion
Similarly, in our recreation of the pipeline, we have not incorporated outlier detection in the ICA part of the pipeline primarily because \citeauthor{cleggRecalibrationMarsScience2017} does not describe their implementation sufficiently enough for replication without unsubstantiated assumptions.
```